### DISCUSSION TITLE
[Feature request]: Connect phone to Apollo server as a gamepad

### DISCUSSION THREAD

**Discussion by FadedAtlas** [ClassicOldSong/moonlight-android#15] [Ideas]
### Is your feature request related to a problem? Please describe.

I would like to sometimes sit behind my PC and use my Razer Kishi (which I'm comfortable with) to use as a gamepad.

### Describe the solution you'd like

Option to connect to Apollo server with just gamepad input. No video stream (audio maybe optional?)

### Describe alternatives you've considered

Downloading other Android apps (client), and bloating my Windows with another background service (server).

**Comment by FadedAtlas** [comment]
Very low priority ofc

**Comment by ClassicOldSong** [comment]
By doing so your phone's screen still need to be on during the game play. I think Kishi can actually connect to PC nativelly.

https://www.reddit.com/r/razer/comments/hkvo5z/kishi_is_pc_compatible_just_need_a_right_angle/

The only other thing you need is 3D printing a rigid block to act like your phone for the controller to clip on.

**Reply by FadedAtlas** [reply]
Yeah, there are workarounds which are working for me rn.
Don’t worry, just wanted to share my idea as a “nice to have.
We probably have more important features to implement and bugs to fix.

Regarding the screen-on issue, we could either display black for OLED screens or request an app wakelock, similar to what Termux does.

And sorry for not posting in "Discussions" from the start, somehow I didn't see them.
